lib/new-admin/transactions.js

```diff
@@ -76,6 +78,46 @@ function batch (from = new Date(0).toISOString(), until = new Date().toISOString
   ]).then(packager)
 }
 
+function simplifiedBatch (from = new Date(0).toISOString(), until = new Date().toISOString(), limit = null, offset = 0) {
+  return batch(from, until, limit, offset)
+    .then(res => {
+      const fields = ['txClass', 'id', 'created', 'machineName',
+        'cryptoCode', 'fiat', 'fiatCode', 'phone', 'toAddress',
+        'txHash', 'dispense', 'error', 'status', 'fiatProfit', 'cryptoAmount']
+      return _.map(_.compose(_.pick(fields), getStatus, getProfit, getCryptoAmount))(res)
+    })
+}
+
+const getCryptoAmount = it => { return { ...it, cryptoAmount: coinUtils.toUnit(BN(it.cryptoAtoms), it.cryptoCode).toString() } }
+
+const getProfit = it => {
+  const getCommissionFee = it => Number.parseFloat(it.commissionPercentage) * Number.parseFloat(it.fiat)
+  if (!it.cashInFee) return { ...it, fiatProfit: getCommissionFee(it) }
+  return { ...it, fiatProfit: getCommissionFee(it) + Number.parseFloat(it.cashInFee) }
+}
+
+const getCashOutStatus = it => {
+  if (it.hasError) return { ...it, status: 'Error' }
+  if (it.dispense) return { ...it, status: 'Success' }
+  if (it.expired) return { ...it, status: 'Expired' }
+  return { ...it, status: 'Pending' }
+}
+
+const getCashInStatus = it => {
+  if (it.operatorCompleted) return { ...it, status: 'Cancelled' }
+  if (it.hasError) return { ...it, status: 'Error' }
+  if (it.sendConfirmed) return { ...it, status: 'Sent' }
+  if (it.expired) return { ...it, status: 'Expired' }
+  return { ...it, status: 'Pending' }
+}
+
+const getStatus = it => {
+  if (it.txClass === 'cashOut') {
+    return getCashOutStatus(it)
+  }
+  return getCashInStatus(it)
+}
```

Comment:
These functions are misleading, we should only return what the name is telling us. e.g. `getStatus` should return only the status, not the entire object